src/sdk/bucketspace_nb.js

```diff
@@ -189,16 +190,19 @@ class BucketSpaceNB {
             _.isUndefined(account.nsfs_account_config.gid)) return false;
         try {
             dbg.log0('_has_access_to_nsfs_dir', namespace_bucket_config.write_resource, account.nsfs_account_config.uid, account.nsfs_account_config.gid);
+            // always allow root access
+            if (account.nsfs_account_config.uid === 0 && account.nsfs_account_config.gid === 0) return true;
```

Comment:
I am not sure about this - it does sound reasonable that root user should see all nsfs buckets, but on the other hand, perhaps there's a more complex ACL on the bucket, and root user is intentionally squashed and not authorized by ACL.
See https://www.ibm.com/docs/ja/spectrum-scale/5.1.0?topic=system-remote-user-access-gpfs-file.
Perhaps we can rethink this check and decide if we really need it, or should simply continue to check access even for root accounts.